### PR TITLE
Image experiment cleanup

### PR DESCRIPTION
**Pull request by oded-soffrin**
- removing unused test experiment
- conver image-viewer to TS

### CODE REVIEW

**Comment by oded-soffrin** [comment]
@mharsat are you proud?

File: packages/plugin-image/web/src/image-viewer.tsx [hunk 1]
```diff
@@ -201,7 +209,13 @@ class ImageViewer extends React.Component {
     );
   };
 
-  getImage(imageClassNames, src, alt, props, opts = {}) {
+  getImage(
+    imageClassNames,
+    src,
+    alt,
+    props,
+    opts: { fadeIn?: boolean; width?: unknown; height?: unknown } = {}
```
Comment (mharsat) [inline packages/plugin-image/web/src/image-viewer.tsx:217]: Use `unknown` sparingly. Sometimes it's hard to figure what exactly is the received type when converting a JS file, but then you can follow its usage until you reach safe TS shores.
In this case it's passed to the `img` HTML element, so we can assume these are in "worst case" `number | string`

**Review by mharsat — CHANGES_REQUESTED** [review]

File: packages/plugin-image/web/src/image-viewer.tsx [hunk 2]
```diff
@@ -22,7 +22,57 @@ import InPluginInput from './InPluginInput';
...
+  dataUrl: string;
+  settings: {
+    onExpand: (blockkey: string) => unknown;
+    disableExpand: boolean;
+    imageProps: () => unknown | Record<string, unknown>;
+  };
+  defaultCaption: string;
+  entityIndex: number;
+  onCaptionChange: () => unknown;
+  setFocusToBlock: () => unknown;
```
[the remark this answers is not in the document]
Comment (oded-soffrin) [reply packages/plugin-image/web/src/image-viewer.tsx:48]: but are they void? I don't know the return value so `unknown` seem appropriate

[the remark this answers is not in the document]
Comment (mharsat) [reply packages/plugin-image/web/src/image-viewer.tsx:48]: Yes you can search where they are used, also `onSomething` functions and `setSomething` are usually void functions

[the remark this answers is not in the document]
Comment (mharsat) [reply packages/plugin-image/web/src/image-viewer.tsx]: This is true for all `void` functions, like `getInPluginEditingMode` and `onExpand` in `ImagePluginViewerConfig`

**Review by mharsat — APPROVED** [review]

**Pull request merged**